deepinv/tests/test_models.py

```diff
@@ -243,34 +242,37 @@ def __init__(self, model, *args, **kwargs):
             self.model = model
 
         def prox(self, x, w):
-            return self.model(x, w)
+            return self.model(x, w[:, 1, :, :].unsqueeze(1))
```

Comment:
Shouldn't it be `w[:, 0, :, :].unsqueeze(1)`? Also, maybe use slicing: `w[:, 0:1, :, :]` without unsqueeze (or `w[:, 1:2, :, :]` in the case where you want channel index 1 and not 0)

Reply:
Well seen, thanks